ui/webui/resources/fonts/download_font.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
when/why does this script run?

Reply:
Manually, so that you can download a new font in the format we want it to be in, with the unicode-range for different locales and separate woff2 files.

The script doesn't strictly need to be in the Chromium repository, but I spent a bit of time reconstructing the process for downloading the font in the right format, and thought it was probably worth including this so the next person doesn't need to.